hspec-core/hspec-core.cabal

```diff
@@ -28,6 +28,11 @@ source-repository head
   location: https://github.com/hspec/hspec
   subdir: hspec-core
 
+flag pretty-printing
```

Comment:
This file is generated. Any changes should go into `package.yaml` instead.

Reply:
Does hpack support `impl` conditionals? I can't see it in its readme.

Reply:
Uh, I was looking at a different `package.yaml`, the one for `hspec-core` already had these conditionals. Fixed.